_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,496 @@
+---
+id: efbfc7
+name: Auto-updating text content can be paused or stopped
+rule_type: atomic
+description: |
+  This rule checks that there are mechanisms to pause or stop the auto-updating of text content
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+acknowledgements:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [visible][] [text node][] in the [flat tree][] of an [HTML document][] if:
+
+- the content of the [text node][] changes through an action that was not initiated by the user;
+- the change happens anytime after the [readiness][document readiness] of the [HTML document][] the [text node][] belongs to is equal to "complete"; and
+- it is not the only [content][] in the [HTML document][].
+
+## Expectation 1
+
+For the test target a [mechanism][] is provided to pause or stop the updating of the content of the [text node][].
+
+**Note**: If there is more than one [text node][] with auto-updating content, a single [mechanism][] may be used to pause or stop updating all [text nodes][text node].
+
+## Expectation 2
+
+The [mechanism][] to pause or stop the auto-updating of the content of the [text node][] is [visible][], has an [accessible name][] that is not empty (""), and is [included in the accessibility tree][].
+
+**Note**: This rule does not require the [mechanism][] to have a [label][] that is descriptive. Depending on how the [mechanism][] is implemented, a non-descriptive label would be a violation of another success criterion, like [Success Criterion 3.3.2: Labels or Instructions][sc 3.3.2]
+
+## Assumptions
+
+This rule assumes that the auto-updating of the content is not [essential][], which is listed as valid exception to [Success Criterion 2.2.2: Pause, Stop, Hide][sc 2.2.2]. When the auto-updating of content is [essential][] this rule may produce incorrect results.
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [Understanding Success Criterion 2.2.2: Pause, Stop, Hide][sc 2.2.2]
+- [G186: Using a control in the Web page that stops moving, blinking, or auto-updating content][g186]
+- [F16: Failure of Success Criterion 2.2.2 due to including scrolling content where movement is not essential to the activity without also including a mechanism to pause and restart the content][f16]
+
+## Test Cases
```

Comment:
All of these examples are inapplicable, because you are using innerText. `innerText` doesn't change the text node, it replaces it.

Reply:
This is addressed by the new applicability